src/customer/closing.py

```diff
@@ -16,7 +16,7 @@
 from itertools import tee
 import json
 
-# Third-Party Libraries
+# cisagov Libraries
 # inter-project packages
```

Comment:
This comment can be deleted, since it is covered by the autogenerated `cisagov Libraries` comment above it.
```suggestion
```

Reply:
Corrected as part of 43ebf41a36b7390834743a1c5d1c711431f8f6f1